src/ftxui/component/renderer.cpp

```diff
@@ -69,6 +62,48 @@ Component Renderer(Component child, std::function<Element()> render) {
   return renderer;
 }
 
+/// @brief Return a focusable component, using |render| to render its interface.
+/// @param render The function drawing the interface, taking a boolean telling
+/// whether the component is focused or not.
+/// @ingroup component
+///
+/// ### Example
+///
+/// ```cpp
+/// auto screen = ScreenInteractive::TerminalOutput();
+/// auto renderer = Renderer([] (bool focused) {
+///   if (focused)
+///     return text("My interface") | inverted;
+///   else
+///     return text("My interface") | inverted;
```

Comment:
This comment doesn't clarify the usage of the focused condition.
Consider differentiating between the two branches with a different decorator.

Reply:
Indeed.
(I pushed two patches together, I will fix this part elsewhere)